### DISCUSSION TITLE
New Slot Settings and Watch

### DISCUSSION THREAD

**Discussion by loopy321** [NightscoutFoundation/xDrip#2024] [Q&A]
Thanks to help from @Navid200, directing me to documentation for proper setting options for "G5/G6 Debug Settings"->"Manual Slot Number"(It was added recently here: [NightscoutFoundation/xDrip@2bc45d6](https://github.com/NightscoutFoundation/xDrip/commit/2bc45d67cb5f636de76cd7b08438eabb1349e44c) )

See link: https://navid200.github.io/xDrip/docs/G6_slot.html

I was able to set Xdrip to "1" and have Spike (on an iPhone) use the phone default "2" (since there is no such setting in that app) and after a few missed readings (maybe interference), I received readings as expected.

Now I am curious if the "Manual Slot" setting gets relayed to the Watch Collector (to use the same slot as set in Xdrip's "Manual Slot Number" (I am using a Sony Smartwatch 3 with the apk extracted and added via adb from the xDrip-plus-20220225-a7e1ef0.apk\res\raw\ file)?  

Thanks.

**Comment by Navid200** [comment]
@jamorham Can the new manual slot setting be used by a watch?

**Reply by funkstille** [reply]
Looking for the exact same feature/ answer to the question from @loopy321

**Reply by tolot27** [reply]
Yes, in general. But this needs a change of the wear apk.

**Reply by funkstille** [reply]
So I do have to compile it myself…
Can you tell me where I need to make the changes?

**Reply by tolot27** [reply]
You just have to repeat the same changes of 2bc45d6 in the corresponding wear files having the same file names.

**Reply by haralha** [reply]
@funkstille did yo make the changes in the wear apk? I can't get slot 1 working on my watch.

**Reply by funkstille** [reply]
To be honest: No, not here in github. 
I am not a developer and also not experienced with github.

I was able to get it working for our watch, at my my offline repo.
But since I am not a skilled  developer this code is not enduser ready (used some hardcoded values).

We are using the watch only for a very special use case (water resistant value collector). 
In general we are using CamAPS and not xDrip.
So since I can not connect xdrip and camaps to dexcom an the same device (bluetooth slots), I have a xdrip mobile phone to just setup the configuration parameters for wear os watch, and never use this phone until the dexcom transmitter ID has to be changed. so every 3 month new transmitter id, and the wear os watch every other weak for 3 hours in the swimming pool

**Comment by xJoe** [comment]
> I was able to set Xdrip to "1" and have Spike (on an iPhone) use the phone default "2" (since there is no such setting in that app) and after a few missed readings (maybe interference), I received readings as expected.

For me it does not work unfortunately. I set xDrip to 1 and start the sensor. At the beginning xDrip gets readings but as soon as I use Shuggah on my iPhone in parallel xDrip looses readings. 

Do you, @loopy321, or others have any ideas what I‘m doing wrong? Thanks.

**Reply by loopy321** [reply]
I had a bunch a problems trying to get it to work stably.  I've moved onto using the O5 pump which takes the slot, so I gave up - sorry.
